src/multiharp_toolkit/ptu_parser.py

```diff
@@ -101,9 +101,9 @@ def parse_record(self, data: int):
     def append_events(self, channel: int, timestamp: float):
         if self.combined_channel:
             self.channels.append(channel)
-            self.timestamps.append(timestamp * 0.2)
+            self.timestamps.append(timestamp * 5)
```

Comment:
I am worried about it would be a MAGIC NUMBER in the future!
You should add some comments, for example, a background formula, principle, purpose, or reference document URL.